Rpackage/R/growtree.R

```diff
@@ -0,0 +1,72 @@
+growtree <- structure(function(target.mat, feature.mat, depth=0, maxdepth = Inf,
+                               margin = 0.0, loss="hinge", id = 1L, min_sample = 1,
+                               pred=NULL, side = NULL, weights = NULL) {
+  
+  pid <- NULL
+  ### creating dataframe of info
+  if(!is.null(pred)){
+    pid <- data.frame(pred[[side]], pred[[1]],row.names = "")
+    colnames(pid) <- c("prediction", "cost")
+  }
+
+  if(side == 10){
```

Comment:
Why 10? We would need a comment to explain what is going on at that line.

Reply:
done